ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -353,6 +373,7 @@ VSFC:
   uses:
   - speed_frequency_command
   - speed_frequency_sensor
+  - speed_percentage_sensor
```

Comment:
Remove, cannot be used here.

Reply:
the field has been deleted in the VSFC abstract.